fondant/component_spec.py

```diff
@@ -38,7 +38,7 @@
 
 
 @dataclass
-class KubeflowInput:
+class Argument:
     """
     Kubeflow component input argument
```

Comment:
```suggestion
    Kubeflow component argument
```
I assume it represents both input and output arguments